plugins/hydra_optuna_sweeper/hydra_plugins/hydra_optuna_sweeper/_impl.py

```diff
@@ -105,6 +109,35 @@ def create_optuna_distribution_from_override(override: Override) -> Any:
 
     raise NotImplementedError(f"{override} is not supported by Optuna sweeper.")
 
+def get_function_from_path(import_string: str) -> Callable[[Trial], None]:
+    method_path = import_string.split(".")
```

Comment:
Not an expert on imports, but I hope this doesn't bring any security issues that we need to take into account (e.g. ../../../etc/passwd). Since we split on '.', I think this is safe?